pkg/xds/dynconf/listener.go

```diff
@@ -18,8 +19,9 @@ import (
 )
 
 const (
-	Origin       = "dynamic-config"
-	ListenerName = "_kuma:dynamicconfig"
+	Origin                       = "dynamic-config"
+	ListenerName                 = "_kuma:dynamicconfig"
+	maxDirectResponseBodyPadding = 256
```

Comment:
nit: but I think this sort of global is just making reading harder. You may as well just define this as a variable in the code, the compiler will simplify the variable out and use a constant anyway.